lib/rubocop/cop/lint/useless_numeric_operation.rb

```diff
@@ -0,0 +1,77 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module Lint
+      # Certain numeric operations have no impact, being:
+      # Adding or subtracting 0, multiplying or dividing by 1 or raising to the power of 1.
+      # These are probably leftover from debugging, or are mistakes.
+      #
+      # @example
+      #
+      #   # bad
+      #   x + 0
+      #   x - 0
+      #   x * 1
+      #   x / 1
+      #   x ** 1
+      #
+      #   # good
+      #   x
+      #
+      #   # bad
+      #   x += 0
+      #   x -= 0
+      #   x *= 1
+      #   x /= 1
+      #   x **= 1
+      #
+      #   # good
+      #   x = x
```

Comment:
Perhaps you meant to write just `x` here, as I don't see how `x = x` makes sense in this context.

Reply:
My thinking was to only remove the numeric component-- the assignment can be handled by `Lint/UselessAssignment`.